app/src/test/java/com/appunite/loudius/ui/reviewers/ReviewersViewModelTest.kt

```diff
@@ -201,32 +264,53 @@ class ReviewersViewModelTest {
 
             viewModel.onAction(ReviewersAction.Notify("user1"))
 
-            assertEquals(ReviewersSnackbarType.SUCCESS, viewModel.state.snackbarTypeShown)
+            expectThat(viewModel.state)
+                .get(ReviewersState::snackbarTypeShown)
+                .isEqualTo(ReviewersSnackbarType.SUCCESS)
         }
 
         @Test
         fun `WHEN successful notify action THEN show loading indicator`() = runTest {
             viewModel = createViewModel()
-            coEvery { repository.notify(any(), any(), any(), any()) } coAnswers { neverCompletingSuspension() }
+            coEvery {
+                repository.notify(
+                    any(),
+                    any(),
+                    any(),
+                    any(),
+                )
+            } coAnswers { neverCompletingSuspension() }
 
             viewModel.onAction(ReviewersAction.Notify("user1"))
 
-            assertTrue(
-                viewModel.state.reviewers.first { it.login == "user1" }.isLoading,
-            ) { "Clicked item should have loading indicator" }
-            assertTrue(
-                viewModel.state.reviewers.filterNot { it.login == "user1" }.none { it.isLoading },
-            ) { "Only clicked item should have loading indicator" }
+            expectThat(viewModel.state) {
+                // Clicked item should have loading indicator
+                get(ReviewersState::reviewers)
+                    .first { it.login == "user1" }
+                    .get(Reviewer::isLoading)
+                    .isTrue()
+
+                // Other items should NOT have loading indicator
+                get(ReviewersState::reviewers)
+                    .filterNot { it.login == "user1" }
+                    .all { get(Reviewer::isLoading).isFalse() }
+            }
         }
 
         @Test
         fun `WHEN failed notify action THEN show failure snackbar`() = runTest {
             every { savedStateHandle.get<String>("pull_request_number") } returns "nonExistingPullRequestNumber"
             viewModel = createViewModel()
 
+            expectThat(viewModel.state)
+                .get(ReviewersState::snackbarTypeShown)
+                .isNull()
+
             viewModel.onAction(ReviewersAction.Notify("user1"))
 
-            assertEquals(ReviewersSnackbarType.FAILURE, viewModel.state.snackbarTypeShown)
+            expectThat(viewModel.state)
+                .get(ReviewersState::snackbarTypeShown)
```

Comment:
**Suggestion** That will fit in one line.

Reply:
as above